### PR TITLE
feat: add elixir app

### PR DESCRIPTION
**Pull request by yordis**
I would love to get the sample app running and see a video including the stats 🙏🏻

### CODE REVIEW

File: lessons/147/elixir-app/deploy/deployment-tls.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+                      - general
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                      - elixir-app
+                      - elixir-app-tls
```
Comment (yordis) [inline lessons/147/elixir-app/deploy/deployment-tls.yaml:58]: Not sure if this is correct

File: lessons/147/elixir-app/deploy/deployment.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+                    values:
+                      - general
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values:
+                      - elixir-app
```
Comment (yordis) [inline lessons/147/elixir-app/deploy/deployment.yaml:50]: Not sure of this is correct

File: lessons/147/elixir-app/deploy/secret-tls.yaml [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: api-elixir-antonputra-pvt
+  namespace: staging
+type: kubernetes.io/tls
+data:
+  tls.crt: |
```
Comment (yordis) [inline lessons/147/elixir-app/deploy/secret-tls.yaml:9]: I copied the `go` certs, so probably they are wrong due to the domain, not sure

**Comment by antonputra** [comment]
I'll take a look

**Comment by yordis** [comment]
@antonputra I need a DigitalOcean version of your setups 😄 AWS is way too complex 😭

**Comment by antonputra** [comment]
I'll create benchmark in the future, I'll close this for now. Thanks!

**Pull request closed without merging**